contrib/runners/remote_runner/remote_runner/runner.yaml

```diff
@@ -6,8 +6,7 @@
   runner_parameters:
     bastion_host:
       description: 'The host SSH connections will be proxied through. Note: This connection
-        is made using the same parameters as the final connection, and is only used
```

Comment:
This description was old and out of date since we don't use fabric runner for a long time anymore.